pyproject.toml

```diff
@@ -14,7 +14,7 @@ requires-python = ">=3.10"
 classifiers = []
 dependencies = [
   "aiohttp>=3.8.1",
-  "ical>=8.3.0",
+  "ical>=10.0.4",
```

Comment:
This needs to be updated once https://github.com/allenporter/ical/pull/517 is released.